src/components/Dashboard/MainContainer/MainContainerUtils.js

```diff
@@ -13,3 +13,21 @@ export const getTaskListAsArray = (snapValue = {}) => {
 
   return taskList;
 };
+
+export const deleteDragOver = (event) => {
+  event.preventDefault();
+  document.getElementById('delete-zone').classList.add('drag-in');
+};
+
+export const deleteDragLeave = (event) => {
+  event.preventDefault();
+  document.getElementById('delete-zone').classList.remove('drag-in');
+};
+
+export const handleDeleteDropCSS = () => {
+  const fakeTask = document.getElementById('drag-avatar');
+  if (fakeTask != null) fakeTask.remove();
+
+  document.getElementById('delete-zone').classList.remove('drag-in');
+  setTimeout(() => document.getElementById('delete-zone').classList.remove('shown'), 150);
```

Comment:
please declare constant for 150 and name it properly, e.g. DELETE_DELAY
it's good practice to avoid magic numbers

Reply:
Thank you!